introduction/architecture.md

```diff
@@ -111,6 +110,68 @@ while maintaining support for multiple indexes.
+single TimescaleDB instances. However, certain actions are blocked on data nodes to protect
+from accidentally executing operations on data nodes that might endanger the data consistency
+of distributed hypertables.
+
+It is important to note that access nodes and data nodes both run TimescaleDB, and for all intents and
+purposes, look just like a single instance of TimescaleDB from an operational perspective.
+
+### Distributed Databases
+
+Distributed databases represent the highest level in the hierarchy used to define a multi-node
+TimescaleDB Cluster. Users can define access nodes and data nodes at the database level. This means
+that if you have multiple databases in a TimescaleDB Cluster, you could theoretically have
+different nodes play both the role of access node and data node for different databases.
+We do not typically suggest running in this manner. Instead, we recommend that you
+only have one access node that acts as a query routing layer between your application and
+all of your backend data nodes.
+
+### Distributed Hypertables
+
+As discussed above, distributed databases contain distributed hypertables. Distributed hypertables are essentially
+the same as regular hypertables, except that they have an additional layer of abstraction where
+access nodes must be able to logically manage distributed hypertables. Distributed hypertables
+are also composed of chunks.
+
+With distributed hypertables, we suggest partitioning the hypertables both by space and by time.
+If you only partition data by time, that chunk will have to fill up before the access node chooses another
+data node to store the next chunk. On the other hand, if you specify a space partition, the access node
+will distribute chunks across multiple data nodes based on the space partition so that multiple
+chunks are created for a given chunk time interval.
+
+We recommend that the number of space partitions are equal or greater than the number of data nodes associated with the distributed hypertable for optimal data distribution across data nodes. The system will automatically increase the number of space partitions, if necessary, when adding new data nodes. In case of multiple space partitions, only the first space partition will be used to determine how chunks are distributed across servers.
+
+### Scaling out with TimescaleDB Clustering
+
+As time-series data grows, a common use case is to add data nodes to expand the storage and compute
+capacity of your TimescaleDB Cluster. TimescaleDB can be scaled out by simply adding data nodes to
+a distributed database.
+
+There are certain cases where you may want to change your partitioning scheme. Because TimescaleDB is
+built specifically for time-series, we leverage the concept of time to provide more flexibility when
+changing partitions. Users can change their partitioning settings, and these new settings will be applied to
+newly created chunks.
```

Comment:
I'm not sure if we've explicitly defined partitioning scheme anywhere, but if not this sounds like we're promising a great deal of flexibility here.  Really the only thing we can really change is the number of partitions (which we'll actually do transparently for users as they scale out).  We can't, for instance, change the column on which we partition.

Reply:
(Agree here with @WireBaron .  I think again the point to emphasize is that, unlike most distributed database, we don't -- and don't need to -- move data between data nodes when cluster is increased, instead just employ a different number of partitions for the *next* time interval.  So the cluster will existing where time intervals before the cluster increase might have 5 partitions over 5 nodes, while after will have 6 partitions over 6 nodes, but the access node is always smart enough to do where to route inserts or queries.

And point out that this works also because as cluster ages, writes to latest time interval continue to be loaded balanced across data nodes in _new_ configuration, rather than older one.

Reply:
Done!